src/components/InfoWindow.js

```diff
@@ -1,11 +1,17 @@
-import React from 'react';
+import React, { useContext } from 'react';
+import { LocationsContext } from '../context/LocationsContext';
 import './InfoWindow.css';
+import getDistanceFromLatLonInKm from '../lib/haversineFunction';
 
 export default function InfoWindow({
   $dimensionKey,
   locationData,
   closeWindow,
 }) {
+  const { coordinates } = useContext(LocationsContext);
+  const reg = new RegExp('^[0-9]*$');
```

Comment:
It's a bit more idiomatic to create your regular expressions as follows:

    const reg = /^[0-9]*$/;

One thing you should change here, though, is to use `+` instead of `*`. Using the asterisk, this will return `true` on an empty string, which I don’t believe you want.

Reply:
thank you!